### DISCUSSION TITLE
想知道1panel 如何配置让php支持http2的

### DISCUSSION THREAD

**Discussion by bin172** [1Panel-dev/1Panel#7561] [General]
想自己用原生Linux安装命令配置php支持 curl http2扩展，我看别的面板都不支持，看到1panel php支持 http2，想知道具体的如何配置的。想学习下具体的安装配置

**Comment by zhengkunwang223** [comment]
> 想自己用原生Linux安装命令配置php支持 curl http2扩展，我看别的面板都不支持，看到1panel php支持 http2，想知道具体的如何配置的。想学习下具体的安装配置

我们没有特殊处理过 镜像都是官方的  所以没有具体的教程

**Reply by bin172** [reply]
这样啊，我自己用原生的装，不是缺这就是缺那
